src/app/clients/getExpandedClientDetails.ts

```diff
@@ -150,15 +157,45 @@ export const formatHouseholdFromFamilyDetails = (
     return `${familyCategory} ${occupantDisplay} (${adultChildBreakdown.join(", ")})`;
 };
 
+export const formatBreakdownOfAdultsFromFamilyDetails = (
+    family: Pick<Schema["families"], "birth_year" | "gender">[]
+): string => {
+    const adultDetails = [];
+
+    for (const familyMember of family) {
+        if (familyMember.birth_year < getCurrentYear() - 16) {
+            const age = getAdultAgeUsingBirthYear(familyMember.birth_year);
+            adultDetails.push(`${age} ${familyMember.gender}`);
+        }
+    }
+
+    if (adultDetails.length === 0) {
+        return "No Adults";
+    }
+
+    return adultDetails.join(", ");
+};
+
 export const formatBreakdownOfChildrenFromFamilyDetails = (
-    family: Pick<Schema["families"], "age" | "gender">[]
+    family: Pick<Schema["families"], "birth_year" | "birth_month" | "gender">[]
 ): string => {
     const childDetails = [];
 
     for (const familyMember of family) {
-        if (familyMember.age !== null && familyMember.age <= 15) {
-            const age = familyMember.age === -1 ? "0-15" : familyMember.age.toString();
-            childDetails.push(`${age}-year-old ${familyMember.gender}`);
+        if (!isAdult(familyMember.birth_year) && !isBaby(familyMember.birth_year)) {
+            const age = getChildAgeUsingBirthYearAndMonth(
+                familyMember.birth_year,
+                familyMember.birth_month,
+                false
+            );
+            childDetails.push(`${age} ${familyMember.gender}`);
+        } else if (isBaby(familyMember.birth_year)) {
+            const age = getChildAgeUsingBirthYearAndMonth(
+                familyMember.birth_year,
+                familyMember.birth_month,
+                false
+            );
+            childDetails.push(`${age} ${familyMember.gender}`);
```

Comment:
The 2 ifs look the same to me? Is it not enough to just have 1 that checks `!isAdult()`? Sorry if I'm missing the difference

Reply:
Yes sorry! I think I handled getting the total months for babies in a separate function before.